pennylane/_qubit_device.py

```diff
@@ -113,32 +113,14 @@ def __init__(self, wires=1, shots=1000, analytic=True):
 
     @classmethod
     def capabilities(cls):
-        """Get the capabilities of the plugin.
 
-        Capabilities include:
-
-        * ``"model"`` (*str*): either ``"qubit"`` or ``"CV"``.
-
-        * ``"inverse_operations"`` (*bool*): ``True`` if the device supports
-          applying the inverse of operations. Operations which should be inverted
-          have ``operation.inverse == True``.
-
-        * ``"tensor_observables" (*bool*): ``True`` if the device supports
-          expectation values/variance/samples of :class:`~.Tensor` observables.
-
-        The qubit device class has built-in support for tensor observables. As a
-        result, devices that inherit from this class automatically
-        have the following items in their capabilities
-        dictionary:
-
-        * ``"model": "qubit"``
-        * ``"tensor_observables": True``
-
-        Returns:
-            dict[str->*]: results
-        """
-        capabilities = cls._capabilities
-        capabilities.update(model="qubit", tensor_observables=True)
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            model="qubit",
+            supports_exact=True,
```

Comment:
Would the idea be that for hardware devices this is explicitly set to `False`? (`IBMQDevice`, `QPUDevice`)

Reply:
You are right, maybe better to have default False?